src/containers/orders/order/order-table/tests/order-table.test.js

```diff
@@ -1,35 +1,134 @@
 import React from 'react';
+import { render, screen, act, fireEvent } from '@testing-library/react';
 import { useDispatch, useSelector } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
 import OrderTable from '../order-table';
+import Modal from '../../../../../components/modal/modal';
 
 jest.mock('../order-table.styles', () => ({
   useStyles: () => ({})
 }));
+jest.mock('../../../../../components/modal/modal.styles', () => ({ useStyles: () => ({}) }));
+jest.mock('../../../cart/cart-item/cart-item.styles', () => ({ useStyles: () => ({}) }));
 jest.mock('react-redux');
 
-const mockCartOperations = { removeFromCart: jest.fn() };
-
 const dispatch = jest.fn();
 useDispatch.mockImplementation(() => dispatch);
+
 const testUseSelector = (lang) => {
   useSelector.mockImplementation(() => ({
-    language: lang
+    currency: 0
   }));
 };
-let wrapper;
 
 const props = {
-  currency: 0,
-  calcPrice: 100,
-  cartLoading: false,
+  cartLoading: true,
   cartQuantityLoading: false,
-  items: [{ product: { product_id: 'some id' }, price: 10 }],
+  items: [
+    {
+      sizeAndPrice: {
+        size: {
+          _id: 'test',
+          name: 'S'
+        },
+        price: [
+          {
+            value: 1000,
+            currency: 'UAH'
+          }
+        ]
+      }
+    }
+  ],
   user: {}
 };
-describe('Order table component tests', () => {
-  it('should match snapshot', () => {
+
+const modalProps = {
+  language: 1,
+  message: 'test',
+  isOpen: true,
+  onAction: jest.fn()
+};
+
+const mockGetCartItem = jest.fn().mockReturnValue({
+  sizeAndPrice: {
+    size: {
+      _id: 'test',
+      name: 'S'
+    },
+    price: [
+      {
+        value: 1000,
+        currency: 'UAH'
+      }
+    ]
+  }
+});
+
+const mockGetTotalPrice = jest.fn();
+const mockChangeQuantity = jest.fn();
+const mockChangeSize = jest.fn();
+
+const mockCartOperations = {
+  getTotalPrice: mockGetTotalPrice,
+  changeQuantity: mockChangeQuantity,
+  getCartItem: mockGetCartItem,
+  changeSize: mockChangeSize,
+  removeFromCart: jest.fn()
+};
+
+describe('test <OrderTable /> component', () => {
+  it('should render <OrderTable />', () => {
     testUseSelector(0);
-    wrapper = shallow(<OrderTable {...props} cartOperations={mockCartOperations} />);
-    expect(wrapper).toMatchSnapshot();
+    const { getByText } = render(
+      <BrowserRouter>
+        <MockedProvider>
```

Comment:
Cant find using of the Mocked provider, and it configuration.

Reply:
fixed